src/views/Course.jsx

```diff
@@ -0,0 +1,17 @@
+import CoursesHero from "../components/Courses/CoursesHero";
+import CourseTextArea from "../components/Course/CourseTextArea";
+import CoreBar from "../components/CoreBar";
+import CoreFooter from "../components/CoreFooter";
+
+function Course() {
+  return (
+    <div>
+   <CoreBar/>
+   <CoursesHero/>
```

Comment:
After reciving props you need to send those probs to children's.

Reply:
figured this out, will need to make a more extensive object but i have the information for the specific course going to the children!